src/en/components/signature/design.md

```diff
@@ -33,16 +53,16 @@ Display the French-first signature on French pages and the English-first on Engl
 
 <img class="b-sm b-default p-300 mb-300" src="/images/en/components/example/example-signature-side-by-side-en.svg" alt="An image presenting both variations of the signature component. The signature type is on the left and the wordmark type is on the right." />
 
-The standard colour style uses black text and a red flag applied to a white backdrop.
+Caption: The standard colour style uses black text and a red flag applied to a white backdrop.
```

Comment:
Are we able to use the `<figure>` and `<figcaption>` elements for these sections?

Reply:
Good call, thanks!